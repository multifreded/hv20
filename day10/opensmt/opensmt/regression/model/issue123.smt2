(set-option :produce-model 1)
(set-logic QF_UF)
(declare-fun b () Bool)
(declare-fun c () Bool)
(assert (= b c ))
(assert (not b))
(check-sat)
(get-value (b c))
