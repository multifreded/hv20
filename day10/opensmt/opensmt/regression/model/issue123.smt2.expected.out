sat
((b false)(c false))
